CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.0.1
```

Comment:
Not a bugfix version but new features one, so it should be 1.1.0 (cf Versioning in README)

Reply:
Oops indeed ^^